.github/workflows/build_orchestrator.yml

```diff
@@ -0,0 +1,57 @@
+name: Build and push Orchestrator images
+
+on:
+  workflow_call:
+  workflow_dispatch:
+
+jobs:
+  build-push:
+    env:
+      IMAGE_NAME: "orchestrator"
+      DOCKERFILE: "Dockerfile.Orchestrator"
+
```

Comment:
I think an improvement in the future would be to have a single workflow for this, and pass in these environment variables. However that could be more general (at a github org level).